db/Seed/workoutSeed.json

```diff
@@ -0,0 +1,802 @@
+[
```

Comment:
OK, so I couldn't force fake data into the Sequelize timestamp hook. Instead, I disabled it and then added a 'created_at' column which I seeded. If that works in your view, this is ready to merge.